### PR TITLE
automatic tendermint-installation

### LINKED ISSUE #450: automated Tendermint installation
Add a script and command for automated tendermint installation, maybe with a check that tendermint is not already installed and also checking its `tendermint version`. This could then be part of `make install`

### PR DESCRIPTION
**Pull request by tzemanovic**
closes #450

### CODE REVIEW

File: scripts/install/get_tendermint.sh [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+export TMP_PATH="/tmp"
+
+error_exit()
+{
+    echo "Error: $1"
+    exit 1
+}
+
+# check for existence
+export TM_EXISTS_VER=$(tendermint version)
```
Comment (tzemanovic) [inline scripts/install/get_tendermint.sh:23]: This fails with `scripts/install/get_tendermint.sh: line 23: tendermint: command not found`, maybe we could silence it

File: scripts/install/get_tendermint.sh [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+export ARCH="amd64"
+if [[ $MACHINE -eq "aarch64" ]]; then
+  ARCH="arm64"
+fi
+
+if [[ $MACHINE -eq "x86_64" ]]; then
+  ARCH="amd64"
+fi
+
+export RELEASE_URL="${TM_REPO}/releases/download/v${TM_VERSION}/tendermint_${TM_VERSION}_${SYSTEM,,}_${ARCH}.tar.gz"
```
Comment (tzemanovic) [inline scripts/install/get_tendermint.sh:42]: I'm getting 
`scripts/install/get_tendermint.sh: line 42: ${TM_REPO}/releases/download/v${TM_VERSION}/tendermint_${TM_VERSION}_${SYSTEM,,}_${ARCH}.tar.gz: bad substitution` here

File: scripts/install/get_tendermint.sh (hunk 1 above)
Comment (atoz7) [reply scripts/install/get_tendermint.sh]: hello tomas,
 
sure ... this problem should be fixed with my latest push to the branch.

br,
joachim

File: scripts/install/get_tendermint.sh (hunk 2 above)
Comment (atoz7) [reply scripts/install/get_tendermint.sh:45]: hello tomas,
 
hmmm ... interesting: what version of bash do you use on your workstation?
so whats the output of `/bin/bash --version`?

additionally: what are the system & architecture strings?
what does `uname -s -m` say?

br,
joachim

Comment (tzemanovic) [reply scripts/install/get_tendermint.sh:45]: hi Joachim, I'm on mac. The first outputs:
```
GNU bash, version 3.2.57(1)-release (x86_64-apple-darwin19)
Copyright (C) 2007 Free Software Foundation, Inc.
```

and the second `Darwin x86_64`

Comment (atoz7) [reply scripts/install/get_tendermint.sh:45]: hello tomas,
 
ok, thank you for the information - i'll look into it :)

br,
joachim

File: scripts/install/get_tendermint.sh [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+error_exit()
+{
+    echo "Error: $1"
+    exit 1
+}
+
+# check for existence
+export TM_EXECUTABLE=$(which tendermint)
+if [ -x ${TM_EXECUTABLE} ]; then
```
Comment (tzemanovic) [inline scripts/install/get_tendermint.sh:24]: In bash 5.0.17, this was still printing out the error. This seems to work:
```suggestion
if [ -x "$TM_EXECUTABLE" ]; then
```

Comment (atoz7) [reply scripts/install/get_tendermint.sh:24]: hello tomas,
 
yes, you are right - just push the change to the feature-branch or should i do it?
 
br,
joachim

Comment (tzemanovic) [reply scripts/install/get_tendermint.sh:24]: I'll commit it from here, thanks!

**Review by tzemanovic — COMMENTED** [review]
LGTM! Tested on mac (bash 3.2.57) and ubuntu (bash 5.0.17)

**Pull request merged**